src/trainingdata/trainingdata.h

```diff
@@ -1,6 +1,6 @@
 /*
```

Comment:
I'm not sure we really want git to have marked trainingdata.h as a fork of writer.h

Reply:
Is it possible that the files in neural being moved to trainingdata were not actually deleted?

Reply:
I just did a straight `git mv`, but git detects that `trainingdata.h` has more in common with the original `writer.h` than the new one. I can do the move as a separate commit, but I expect it will still show it like that in the next step.